pages/mission.vue

```diff
@@ -0,0 +1,249 @@
+<template>
+  <div id="page" ref="content" class="toPdf">
+    <div id="left--part">
+      <div id="image">
```

Comment:
id name "image" might be a bit too simple ? A more complex name could be better for an Id ?